bin/reth/src/node/mod.rs

```diff
@@ -87,6 +87,11 @@ use std::{
 use tokio::sync::{mpsc::unbounded_channel, oneshot, watch};
 use tracing::*;
 
+#[cfg(feature = "optimism")]
+use reth_rpc_api::EngineApiClient;
+#[cfg(feature = "optimism")]
+use reth_rpc_types::engine::ForkchoiceState;
```

Comment:
let's get rid of those imports and use full path when we're using these types instead

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/145